grants_tagger/predict_mesh.py

```diff
@@ -0,0 +1,37 @@
+"""
+Predict function for disease part of mesh that optionally
+exposes probabilities and that you can set the threshold 
+for making a prediction
+"""
+import pickle
+import os
+
+from numpy import hstack
+
+
+def predict_mesh_tags(X, model_path, label_binarizer_path,
+                      probabilities=False, threshold=0.5):
+    # TODO: generalise tfidf to vectorizer.pkl
+    with open(f"{model_path}/tfidf.pkl", "rb") as f:
+        vectorizer = pickle.loads(f.read())
+    with open(label_binarizer_path, "rb") as f:
+        label_binarizer = pickle.loads(f.read())
+
+    nb_labels = len(label_binarizer.classes_)
+    Y_pred_proba = []
+    for tag_i in range(0, nb_labels, 512):
```

Comment:
Is 512 a default batch size? In this case you could perhaps get `y_batch_size` as an argument in line with your train function (https://github.com/wellcometrust/grants_tagger/blob/50b2f09011c3f5475a6af5fccb58dd566ddb3452/grants_tagger/train.py#L186)

Reply:
indeed, good point

Reply:
8ce1ab896ba8c021dc9fd3ee7b90fb3c32a95982